ChemoTracker/components/Login/Login.js

```diff
@@ -7,15 +7,12 @@ import styles from '../../styles/login_screen.js';
 
 export default class Login extends React.Component {
 
-  capi = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
-
-
-
   state = {
```

Comment:
add this inside a constructor (look at Calendar.js)

Reply:
why is it necessary? i dont need anything from the parent

Reply:
It doesn't really matter I guess